packages/next-rspack/index.js

```diff
@@ -1,11 +1,16 @@
+Error.stackTraceLimit = 100
 module.exports = function withRspack(config) {
-  process.env.NEXT_RSPACK = 'true'
-  process.env.RSPACK_CONFIG_VALIDATE = 'loose-silent'
+  if (process.env.NEXT_RSPACK === 'true') {
```

Comment:
this is a race condition given that we load the config on the main thread and in a worker thread.  @bgw how scared should i be